docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -118,6 +123,7 @@ The following is a sample JSON file.
 
 The YAML file structure is similar to that of a JSON. The key-value pair of connection ID and the definitions of one or more connections.
 The connection can be defined as a URI (string) or JSON object.
+
 For a guide about defining a connection as a URI, see:: :ref:`generating_connection_uri`.
 For a description of the connection object parameters see :class:`~airflow.models.connection.Connection`.
```

Comment:
This content is unnecessarily duplicated.

Reply:
Dropped